### PR TITLE
feat: tvm for clear gate cellars

### PR DESCRIPTION
**Pull request by codingki**
## Description

TVM for clear gate cellars using the same method with aave, We move it to `/common`

## Changes

- [x] [feat: tvm for clear gate cellars](https://github.com/strangelove-ventures/sommelier/commit/2c5584eef0b7dbdb65df2944e55bb40a7f4799be)

### CODE REVIEW

File: src/data/hooks/useTvm.ts [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 import { useCreateContracts } from "./useCreateContracts"
 
 export const useTvm = (config: ConfigProps) => {
   const { cellarContract } = useCreateContracts(config)
 
   const [cellarResult] = useGetCellarQuery({
     variables: {
-      cellarAddress: config.id,
-      cellarString: config.id,
+      cellarAddress: config.id.toLowerCase(),
```
Comment (sandbochs) [inline src/data/hooks/useTvm.ts:12]: should we just lowercase all the addresses in the config?

Comment (codingki) [reply src/data/hooks/useTvm.ts:12]: make sense, will change that

Comment (codingki) [reply src/data/hooks/useTvm.ts]: solved in https://github.com/strangelove-ventures/sommelier/pull/537

**Review by xanderjl — APPROVED** [review]
LGTM 💪

**Review by sandbochs — APPROVED** [review]

**Pull request merged**